.nvmrc

```diff
@@ -0,0 +1 @@
+12.22.7
```

Comment:
I think you mean 12.22.10 right?

Reply:
Argh I copied the file from selfservice but hadn't updated the codebase

Reply:
fixed